packages/ui/index.ts

```diff
@@ -7,6 +7,7 @@ import Link from '@littlespoon/link'
 import Tag from '@littlespoon/tag'
 import theme from '@littlespoon/theme'
 import Typography from '@littlespoon/typography'
+import Label from 'label'
```

Comment:
This should be `@littlespoon/label`:

```suggestion
import Label from '@littlespoon/label'
```

Reply:
Updated